docs/support.md

```diff
@@ -68,6 +68,28 @@ Select **[Request to reset Multi Factor Authentication]** for issue option and f
 #### Step 2
 Sign in to TechPass Portal and follow the series of prompt to set up MFA in your authenticator app using your new mobile device
 
+## Trouble Signing In - Check Device Status in DEEP Portal
+
+DEEP is a system that helps developers establish a robust security baseline for their devices, while ensuring only compliant devices can access Government engineering resources.
+
+### Protecting developer devices
+DEEP applies a security configuration baseline for each developer device based on industry standards such as the CIS benchmark. It also alerts developers on configuration and malware-related issues via the DEEP Dashboard, providing detailed remediation instructions for each issue.
+
+### Protecting Government engineering resources
+DEEP makes use of device-specific configuration and malware information to block developers with at-risk devices from accessing Government engineering resources.
+
+### When a user is blocked by DEEP at the Cloudflare level, the user will be presented this error:
+![mdm_cloudflare_error](assets/support/mdmCloudflareError.png)
+
+### When the user has established a connection with Cloudflare and the user's device has been blocked by DEEP, the user will be get this error:
+![mdm_techpass_error](assets/support/mdmTechPassError.png)
+
+### DEEP Portal Links
```

Comment:
I was testing the deep link: https://prod.dashboard.dso360.tech
Looks like i can't access it via macbook.
1. can you provide the instructions to tell user to visit deep's prod env via gsib and follow instructions to resolve their mdm compliance issues. then once resolve ask them to test techpass access again (be it gsib/macbook)?
2. is there any links to staging env?

basically we need to list down the different routes/conditions where the user can encounter this error.

Reply:
https://prod.dashboard.dso360.tech will be accessible only if you are participating in the MDM end-to-end testing in PROD.

1. the portal is actually accessible via the work laptop (MacBook and Windows) and that's also the device that's being managed.
2. there's no link to staging yet but will be available once they have finished VAPT.